test/test_slave.html

```diff
@@ -18,7 +18,14 @@
   <head>
     <title>PDF.js test slave</title>
     <meta charset="utf-8">
-    <script src="../build/generic/build/pdf.js"></script>
+    <script>
+    if (location.search.replace('?', '&').includes('&coverage=true')) {
+      // document.write(`<script src="../coverage/lcov-report/pdf.js/build/generic/build/pdf.js"><\/script>`);
+      document.write(`<script src="../build/generic/build/pdf.js"><\/script>`);
+    } else {
+      document.write('<script src="../src/pdf.js"><\/script>');
+    }
+    </script>
     <script src="../build/components/pdf_viewer.js"></script>
```

Comment:
I guess something is up with the paths specified in these `script` statements. Could you please check them again.

Initially they were set to 
```
<script>
    if (location.search.replace('?', '&').includes('&coverage=true')) {
      document.write(`<script src="../coverage/lcov-report/pdf.js/build/generic/build/pdf.js"><\/script>`);
    } else {
      document.write(`<script src="../build/generic/build/pdf.js"><\/script>`);
    }
    </script>
```
and they were not working because the path never existed in the first place.



I've changed it to the one present above but still no JSON data is being received.